sites/cheerpx/src/content/docs/13-tutorials/setup-svelte.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Svelte setup
```

Comment:
Let's reframe this as using CheerpX with NPM. Svelte is just an example for this larger topic.